languages/go/exercises/concept/conditionals/.docs/hints.md

```diff
@@ -6,48 +6,62 @@ Conditionals are used to check for certain conditions and/or criteria. The most
 
 ### 1. Multiple cases (else if)
 
-For various cases an `else if` and `else` statements can be used like this: 
+For various cases an `else if` and `else` statements can be used like this:
 ```go
 if condition {
-    // some code
+    // conditional code
 } else if other_condition {
-    // some code
-} else if another_condition {
-    // some code
+    // conditional code
 } else {
-    // some code
+    // default code
 }
```

Comment:
I'm a little bit unsure of having code in the `hints.md` file. The [docs state](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd)

> The hints should not spell out the solution, but instead point to a resource describing the solution

So maybe instead of having the code here, you could point to an external resource? Would that be possible?

Reply:
Yes, will look into it. Will move the code samples to the introduction and adjust them.

Reply:
Awesome!

Reply:
Rewritten the hints to be actual hints.